cli.js

```diff
@@ -41,13 +41,13 @@ if (args['--help'] || !args._.length) {
     process.exit(0)
 }
 
-const [command] = args._
+const [command, workdir = '.'] = args._
 
 if (commands.includes(command)) {
     const result = spawn.sync(
         'node',
         ['-r', 'esm', require.resolve(`./commands/${command}`)],
-        { stdio: 'inherit' }
+        { env: { ...process.env, WORKDIR: workdir }, stdio: 'inherit' }
```

Comment:
We could've overridden `cwd`, but the original `cwd` is used to check for `tsconfig.json` and `yarn.lock`